src/SlmQueueBeanstalkd/Queue/BeanstalkdQueue.php

```diff
@@ -26,9 +31,13 @@ class BeanstalkdQueue extends AbstractQueue implements BeanstalkdQueueInterface
      * @param string           $name
      * @param JobPluginManager $jobPluginManager
      */
-    public function __construct(Pheanstalk $pheanstalk, $name, JobPluginManager $jobPluginManager)
+    public function __construct(Pheanstalk $pheanstalk, $name, JobPluginManager $jobPluginManager, QueueOptions $options = null)
```

Comment:
This line is also causing the errors in Travis btw: https://travis-ci.org/juriansluiman/SlmQueueBeanstalkd/jobs/72837356#L370. Can we make it non optional, which removes 7 chars from the line. That's still 1 too long for PSR-2 :sob: but split it over multiple lines and all is mergeable!